### PR TITLE
#624 Reworked About Us

### LINKED ISSUE #624: About us Section
## Brief description:
Rework the About us section, the design can be found here: [HackAUBG website design](https://www.figma.com/proto/i6irsDkQVe27tYzzDrPiwb/Hub-stickers?page-id=65:505&type=design&node-id=75-5137&viewport=1188,-1373,0.43&t=BAIKVgSp9EDxjRzs-1&scaling=min-zoom)
## How to achieve it:
Open the hackAUBG folder find the About us section and edit the design. Make sure it is responsive by adding/fixing media queries if any issues occur on mobile.

Experiment making a background similar to the design by using svg websites like: [SVG Generator website](https://www.figma.com/proto/i6irsDkQVe27tYzzDrPiwb/Hub-stickers?page-id=65:505&type=design&node-id=75-5137&viewport=1188,-1373,0.43&t=BAIKVgSp9EDxjRzs-1&scaling=min-zoom)

### PR DESCRIPTION
**Pull request by Delenil**
#624  About Us Section

-Reworked About Us Section to adhere to the new design - (new font, Background).
-Reworked Video Section to adhere to new design - (new font, background, box shadow).
-Cleared [background: linear-gradient(#204829, #222222 50%)] from Schedule Section, FAQ Section, and Video Section.

Resolves #624

### CODE REVIEW

File: packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
     padding-bottom: 3%;
     display: flex;
     flex-direction: column;
-    background: linear-gradient(#222222, #204829);
+    /* background: linear-gradient(#222222, #204829); */
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css:5]: Remove all commented code

File: packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css [hunk 2]
```diff
@@ -1,26 +1,38 @@
...
+    /* background-image: linear-gradient(rgba(150, 0, 0, 0), #204829); */
     justify-content: center;
     align-items: center;
     display: flex;
     flex-direction: column;
 }
 
 .video-header {
-    width: 70%;
+    width: 1200px;
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css:17]: Hmm. Hardcoded value don't look good to me. Is this responsive? @Aleks1908

File: packages/web/src/components/spa/HackAUBG/hack_aubg.css [hunk 3]
```diff
@@ -22,13 +22,5 @@
...
-    background-image: linear-gradient(
-        to bottom,
-        #222222,
-        #222222,
-        #222222,
-        #222222,
-        #222222,
-        #222222
-    );
+    background-image: linear-gradient(to bottom, #0a0a0a, #0a0a0a, #0a0a0a, #0a0a0a, #0a0a0a, #0a0a0a);
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/hack_aubg.css:25]: Just change to background-color: #0a0a0a

**Review by Aleks1908 — APPROVED** [review]
LGTM

**Pull request merged**